<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TheoreticalSpectrumGenerationDialogTemplate</class>
 <widget class="QDialog" name="TheoreticalSpectrumGenerationDialogTemplate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate theoretical spectrum</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="enter_seq_label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Enter sequence:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="seq_type">
         <item>
          <property name="text">
           <string>Peptide</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RNA</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="seq_input">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="charge_label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>35</height>
                </size>
               </property>
               <property name="text">
                <string>Charge:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="charge_spinbox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="minimum">
                <number>-99</number>
               </property>
               <property name="value">
                <number>1</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QGroupBox" name="isotope_model">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>90</height>
              </size>
             </property>
             <property name="title">
              <string>Isotope Model:</string>
             </property>
             <widget class="QRadioButton" name="model_none_button">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>20</y>
                <width>91</width>
                <height>22</height>
               </rect>
              </property>
              <property name="text">
               <string>None</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="QRadioButton" name="model_coarse_button">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>40</y>
                <width>91</width>
                <height>22</height>
               </rect>
              </property>
              <property name="text">
               <string>Coarse</string>
              </property>
             </widget>
             <widget class="QRadioButton" name="model_fine_button">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>60</y>
                <width>91</width>
                <height>22</height>
               </rect>
              </property>
              <property name="text">
               <string>Fine</string>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_16">
               <item>
                <widget class="QLabel" name="max_iso_label">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777213</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="whatsThis">
                  <string>Defines the maximal isotopic peak which is added</string>
                 </property>
                 <property name="text">
                  <string>Max. Isotope:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="max_iso_spinbox">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="value">
                  <number>2</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_17">
               <item>
                <widget class="QLabel" name="max_iso_prob_label">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Max. Isotope
Probability in %</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="max_iso_prob_spinbox">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="readOnly">
                  <bool>false</bool>
                 </property>
                 <property name="value">
                  <number>5</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="generate_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Generate:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="ion_types">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>155</height>
            </size>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <item>
            <property name="text">
             <string>A-ions</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A-b-ions</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B-ions</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>C-ions</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>D-ions</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>W-ions</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>X-ions</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Y-ions</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Z-ions</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Precursor</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Neutral losses</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Abundant immonium-ions</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="intensities">
       <property name="title">
        <string>Intensities</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="a_label">
            <property name="text">
             <string>A-ions</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="a_intensity">
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="a_b_label">
            <property name="text">
             <string>A-b-ions</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="a_b_intensity">
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="b_label">
            <property name="text">
             <string>B-ions</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="b_intensity">
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="c_label">
            <property name="text">
             <string>C-ions</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="c_intensity">
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="d_label">
            <property name="text">
             <string>D-ions</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="d_intensity">
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="w_label">
            <property name="text">
             <string>W-ions</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="w_intensity">
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="x_label">
            <property name="text">
             <string>X-ions</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="x_intensity">
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="y_label">
            <property name="text">
             <string>Y-ions</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="y_intensity">
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="z_label">
            <property name="text">
             <string>Z-ions</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="z_intensity">
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="rel_loss_label">
            <property name="text">
             <string>Relative loss in %</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="rel_loss_intensity">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialog_buttons">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dialog_buttons</sender>
   <signal>rejected()</signal>
   <receiver>TheoreticalSpectrumGenerationDialogTemplate</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
